CharacterCreation/01_00_Species_Overview.txt

```diff
@@ -36,6 +36,22 @@ Species Overview
 
 ===== Humans =====
 
+    Humans are bipedal organic mammals. They origionally hail from a formerly
+blue-green planet called Earth, and now make up the majority of the Terran 
+Empire. Humans are hearty, enterprising and resourceful. 
+
+    Humans can excell at any class. You could be a grizzled human soldier who's
```

Comment:
Sp: Excel
Gr: as any class
Suggestion: "who has" has better cadence and clearer meaning than "who's." (Who's most often refers to "who is.")